core/src/test/java/com/graphhopper/reader/dem/AbstractEdgeElevationInterpolatorTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper GmbH licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.reader.dem;
+
+import java.util.Arrays;
+
+import org.junit.After;
+import org.junit.Before;
+
+import com.graphhopper.coll.GHBitSetImpl;
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.util.DataFlagEncoder;
+import com.graphhopper.routing.util.EncodingManager;
+import com.graphhopper.routing.util.FootFlagEncoder;
+import com.graphhopper.storage.GraphExtension;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.storage.RAMDirectory;
+import com.graphhopper.util.EdgeIteratorState;
+
+import gnu.trove.set.hash.TIntHashSet;
+
+/**
+ * @author Alexey Valikov
+ */
+public abstract class AbstractEdgeElevationInterpolatorTest {
```

Comment:
Ah, ok then you can keep it. Maybe this changed in some more recent maven version.